core/src/main/java/org/apache/gravitino/authorization/AuthorizationUtils.java

```diff
@@ -255,15 +245,33 @@ public static void authorizationPluginRemovePrivileges(
     if (GravitinoEnv.getInstance().accessControlDispatcher() != null) {
       MetadataObject metadataObject = NameIdentifierUtil.toMetadataObject(ident, type);
       MetadataObjectChange removeObject = MetadataObjectChange.remove(metadataObject);
+
+      String metalake =
+          type == Entity.EntityType.METALAKE ? ident.name() : ident.namespace().level(0);
+
       callAuthorizationPluginForMetadataObject(
-          ident.namespace().level(0),
+          metalake,
           metadataObject,
           authorizationPlugin -> {
             authorizationPlugin.onMetadataUpdated(removeObject);
           });
     }
   }
 
+  public static void removeCatalogPrivileges(Catalog catalog) {
```

Comment:
There seems no need to extract this logic as a method.
Thoughts?

Reply:
I prefer putting the logic into `AuthorizationUtils`.

Reply:
Em... then why don't we publicize this `callAuthorizationPluginImpl`?